test/e2e/storage/framework/testdriver.go

```diff
@@ -166,6 +166,7 @@ const (
 	CapRWX                 Capability = "RWX"                 // support ReadWriteMany access modes
 	CapControllerExpansion Capability = "controllerExpansion" // support volume expansion for controller
 	CapNodeExpansion       Capability = "nodeExpansion"       // support volume expansion for node
+	CapOfflineExpansion    Capability = "offlineExpansion"    // supports offline volume expansion
 	CapOnlineExpansion     Capability = "onlineExpansion"     // supports online volume expansion
```

Comment:
There should be some comment about defaulting of these capabilities in the comment.

Reply:
Updated, thanks for the review!